docs/application/design/wearable/ui-components/list.md

```diff
@@ -70,7 +67,7 @@ You can design list pages using one of the following formats. The default height
 
 
 | ![](media/ui_components_10.2.4_1-850x174_1.png) | ![](media/ui_components_10.2.4_1-850x174_2.png) | ![](media/ui_components_10.2.4_1-850x174_3.png) |
-|-----|-----|-----|
+| -- | -- | -- |
```

Comment:
simple Question : Why spaces are added here?

Reply:
No problem. | -- | and |----| are same.